nvm.sh

```diff
@@ -3679,11 +3852,31 @@ nvm() {
         nvm_sanitize_path nvm_has_colors nvm_process_parameters \
         nvm_node_version_has_solaris_binary nvm_iojs_version_has_solaris_binary \
         nvm_curl_libz_support nvm_command_info nvm_is_zsh nvm_stdout_is_terminal \
+        nvm_get_colors nvm_set_colors nvm_print_color_code \
+        nvm_echo_with_colors nvm_err_with_colors \
         >/dev/null 2>&1
       unset NVM_RC_VERSION NVM_NODEJS_ORG_MIRROR NVM_IOJS_ORG_MIRROR NVM_DIR \
-        NVM_CD_FLAGS NVM_BIN NVM_INC NVM_MAKE_JOBS \
+        NVM_CD_FLAGS NVM_BIN NVM_INC NVM_MAKE_JOBS NVM_COLORS NVM_RED_INFO NVM_GREEN_INFO \
+        NVM_BLUE_INFO NVM_CYAN_INFO NVM_MAGENTA_INFO NVM_YELLOW_INFO NVM_BLACK_INFO \
+        NVM_GREY_WHITE_INFO NVM_INSTALLED_COLOR NVM_SYSTEM_COLOR NVM_CURRENT_COLOR \
+        NVM_NOT_INSTALLED_COLOR NVM_DEFAULT_COLOR NVM_LTS_COLOR NVM_INITIAL_COLOR_INFO \
```

Comment:
can we find a way to avoid all the `NVM_*_INFO` vars here?

Reply:
The info each contains could be set directly within the help message text, and the logic for whether the color code is displayed in color or plain text would move to the lines starting near 2600. This might be a little harder to read, but eliminates the variables.

Can you see any issues arising from this?

Reply:
We can create local variables inside the help-printing code to help; it seems like a fine approach to me.

Reply:
Since the variables are already set at the top of that help-printing section near line 2510, declaring them as local and then assigning the values seems to be a more immediate solution.

Reply:
Ok, this is changed.